src/vue/backend/src/templates/ListingsAdvanced.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <idx-block tag="fieldset" className="fieldset listings-advanced">
+        <idx-form-group>
+            <idx-block className="control-label"><strong>CSS Settings</strong> Detailed sentence or two describing deregistering IMPress Listing CSS files so that the installed theme’s CSS won’t have specificity issues.</idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-toggle-slider">
+                {{ mainCssLabel }}
+                <idx-toggle-slider
+                    uncheckedState="No"
+                    checkedState="Yes"
+                    @toggle="listingsSettingsStateChange({ key: 'deregisterMainCss', value: !deregisterMainCss })"
+                    :active="deregisterMainCss"
+                    :label="mainCssLabel"
+                ></idx-toggle-slider>
+            </idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-toggle-slider">
+                {{ widgetsCssLabel }}
+                <idx-toggle-slider
+                    uncheckedState="No"
+                    checkedState="Yes"
+                    @toggle="listingsSettingsStateChange({ key: 'deregisterWidgetCss', value: !deregisterWidgetCss })"
+                    :active="deregisterWidgetCss"
+                    :label="widgetsCssLabel"
+                ></idx-toggle-slider>
+            </idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-label"><strong>Form Submissions to IDX Broker</strong> Send all contact form submissions to IDX Broker as a lead.<br> Note: This option only works while using default contact forms.</idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-toggle-slider">
+                {{ sendFormLabel }}
+                <idx-toggle-slider
+                    uncheckedState="No"
+                    checkedState="Yes"
+                    @toggle="listingsSettingsStateChange({ key: 'sendFormSubmission', value: !sendFormSubmission })"
+                    :active="sendFormSubmission"
+                    :label="sendFormLabel"
+                ></idx-toggle-slider>
+            </idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-label"><strong>Default Form Shortcode</strong> Detailed sentence or two describing short code compatibility with Contact Form plugin. If no short code is entered the template uses default contact forms.</idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-form-label customClass="control-label" for="form-shortcode">Form Shortcode</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="form-shortcode"
+                :value="formShortcode"
+                @change="listingsSettingsStateChange({ key: formShortcode, value: $event.target.value })"
```

Comment:
Key needs to be a string, otherwise it does not update the state correctly. Same on lines 66, 78, and 87